projects/core/src/button/button.element.ts

```diff
@@ -83,6 +83,21 @@ export class CdsButton extends CdsBaseButton {
   @property({ type: String })
   loadingState: keyof typeof ClrLoadingState = ClrLoadingState.default;
 
+  // when the loading state changes, the disabled state should be set to what the consumer manually set, if set
+  // the setter here should never be called in the component, call super instead
+  //  https://github.com/vmware-clarity/core/issues/129
+  private _disabled_externally = false;
```

Comment:
Snake case is usually not used in JS. So I think this should renamed to `_disabledExternally`.